### PR TITLE
swaps: fix fast currency select row

### PR DESCRIPTION
**Pull request by skylarbarrera**
## What changed (plus any additional context for devs)
for output tokens we don't use the account asset so having it override the passed item is not gucci. I changed this and fixed the props for token icon to support L2 tokens 

I fixed some selector and prop ordering in the exchange list as well

not sure if my style on the info button is good for a fast component, lmk what should be changed if not

## Screen recordings / screenshots

https://cloud.skylarbarrera.com/Screen-Recording-2022-07-15-11-26-39.mp4

## What to test

check swaps + discover

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [x] Did you test both iOS and Android?
- [x] If your changes are visual, did you check both the light and dark themes?
- [x] Added e2e tests? If not, please specify why
- [x] If you added new critical path files, did you update the CODEOWNERS file?
- [x] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

File: src/components/asset-list/RecyclerAssetList2/FastComponents/FastCurrencySelectionRow.tsx [hunk 1]
```diff
@@ -178,6 +180,7 @@ export default React.memo(function FastCurrencySelectionRow({
               useActionSheetFallback={false}
               wrapNativeComponent={false}
               {...contextMenuProps}
+              {...((showFavoriteButton || showAddButton) && { style: sx.info })}
```
Comment (terrysahaidak) [inline src/components/asset-list/RecyclerAssetList2/FastComponents/FastCurrencySelectionRow.tsx:183]: This should be just
```
style={(showFavoriteButton || showAddButton) && sx.info}
```

**Review by terrysahaidak — APPROVED** [review]
Overall looks good

**Review by estebanmino — APPROVED** [review]
:chef-kiss:

**Pull request merged**